lib/active_model_serializers/model.rb

```diff
@@ -6,7 +6,36 @@ class Model
     include ActiveModel::Serializers::JSON
     include ActiveModel::Model
 
-    class_attribute :attribute_names
+    # Configuration to avoid a breaking change with older versions of this class which lacked defined attributes.
+    # Previous behavior was: 1) initialized attributes were the
+    class_attribute :attributes_are_always_the_initialization_data, instance_writer: false, instance_reader: false
+    self.attributes_are_always_the_initialization_data = false # remove this commit, just for demonstration
```

Comment:
sorry to be unclear, I just added [Hey look it passes](bba724c211384f13682323d15424afb7e56ad63d) to show that I didn't break the existing test suite when 'false'. I'm going to remove that commit :)